lib/beaker/host/pswindows.rb

```diff
@@ -23,5 +23,17 @@ def platform_defaults
       })
     end
 
+    attr_reader :network_separator, :external_copy_base, :system_temp_path
+    def initialize name, host_hash, options
+      super
+
+      @network_separator = '/'
+      @external_copy_base = '/programdata'
+      # %TEMP% == C:\Users\ADMINI~1\AppData\Local\Temp
+      # is a user temp path, not the system path.  Also, it doesn't work, there's
+      # probably an issue with the `ADMINI~1` section
+      @system_temp_path = 'C:\\Windows\\Temp'
```

Comment:
Looks like we previously hardcoded this, but if we could avoid that, it would be better.